api/src/cli/utils/create-env/env-stub.liquid

```diff
@@ -1,60 +1,331 @@
 ####################################################################################################
-## General
+# 
+# These values set variables which modify core settings of Directus. The variables can be stored
+# directly in your environment (which means the variables are accessible from the operating system)
+#
+# It is more performant to set these values in your environment directly rather than in this file
```

Comment:
How is it more performant? I'm also wondering if that's the case for every environment you can run Directus in in the first place 🤔

Reply:
I read somewhere in the symfony community  that env vars should be added to the system directly in a production environment rather than keeping them in a file due to performance reasons. this may have been in irc or the symfony discord. 

Whether this applies to all the systems like docker and so on, I'd imagine so, but from a quick duckduckgo search I can't find anything. I did find that for docker, you can add the env vars during image build: https://vsupalov.com/docker-build-time-env-values/

Reply:
The reason for not keeping vars in files with Symfony is that PHP will read the file on every request. Directus runs as a service and will only read the config file once when the service is started.

Reply:
Oh, good point. So the performance boost would be negligible.